pkg/controller/services.go

```diff
@@ -307,6 +307,37 @@ func apicRedirectPol(name string, tenantName string, nodes []string,
 	return rp, rpDn
 }
 
+func apicRedirectPolSnat(name string, tenantName string, nodes []string,
+	nodeMap map[string]*metadata.ServiceEndpoint,
+	monPolDn string, healthGroupDnMap map[string]string) (apicapi.ApicObject, string) {
+
+	rp := apicapi.NewVnsSvcRedirectPol(tenantName, name)
+	rp.SetAttr("thresholdDownAction", "deny")
+	rpDn := rp.GetDn()
+	for _, node := range nodes {
```

Comment:
I am wondering why we need to range over a nodes slice when we have two maps with nodes as keys. Can't we just range over one of those maps instead?

Reply:
The reason to range over nodes slice is because healthGroupDn and ServiceEP are per node.